Controllers/AuthenticationController.js

```diff
@@ -1,6 +1,7 @@
 import User from '../Models/User.js'
 import bcrypt from 'bcrypt'
 import { validationResult } from 'express-validator'
+import UserConfig from '../Models/UserConfig.js';
```

Comment:
Please move the user config into the user model.... it should should just be 'configs'